cmd/kpod/history.go

```diff
@@ -83,226 +94,183 @@ func historyCmd(c *cli.Context) error {
 		return err
 	}
 
+	format := genHistoryFormat(c.Bool("quiet"))
+	if c.IsSet("format") {
+		format = c.String("format")
+	}
 	human := true
 	if c.IsSet("human") {
 		human = c.Bool("human")
 	}
 
-	truncate := true
-	if c.IsSet("no-trunc") {
-		truncate = !c.Bool("no-trunc")
-	}
-	quiet := false
-	if c.IsSet("quiet") {
-		quiet = c.Bool("quiet")
-	}
-	format := ""
-	if c.IsSet("format") {
-		format = c.String("format")
-	}
 	args := c.Args()
 	if len(args) == 0 {
-		logrus.Errorf("an image name must be specified")
-		return nil
+		return errors.Errorf("an image name must be specified")
 	}
 	if len(args) > 1 {
-		logrus.Errorf("Kpod history takes at most 1 argument")
-		return nil
+		return errors.Errorf("Kpod history takes at most 1 argument")
 	}
 	imgName := args[0]
 
 	opts := historyOptions{
-		image:    imgName,
-		human:    human,
-		truncate: truncate,
-		quiet:    quiet,
-		format:   format,
+		image:   imgName,
+		human:   human,
```

Comment:
If you are cleaning this up, lets get rid of the temporary variables. Any reason to have human?

Reply:
Yes, human needs to be set to true by default.

Reply:
You can set Human to true by using a BoolTFlag

Reply:
ooh, TIL :)